ocular/src/chain/client/automated_tx_handler.rs

```diff
@@ -521,13 +522,10 @@ mod tests {
         // Execute on toml; expect tx error, but ONLY tx error, everything else should work fine. Tx fails b/c this is unit test so no network connectivity
         let err = chain_client
             .execute_delegated_transacton_toml(toml_save_path, false)
-            .await
-            .err()
-            .unwrap()
-            .to_string();
+            .await;
 
-        // Expect Tx error b/c unit test has no network connectivity; do string matching b/c exact error type matching is messy
-        assert_eq!(&err[..35], "chain client error: transport error");
```

Comment:
Had to clean up these tests here; unrelated to caching PR, they were just in a broken state, error messages aren't always consistent, hence the change.